packages/devtools_app/lib/src/shared/theme.dart

```diff
@@ -554,7 +554,7 @@ double get chartFontSizeSmall => scaleByFontFactor(12.0);
 const lightSelection = Color(0xFFD4D7DA);
 
 bool includeText(
-    BuildContext context, double minScreenWidthForTextBeforeScaling) {
+    BuildContext context, double? minScreenWidthForTextBeforeScaling) {
```

Comment:
nit: not your fault but add a trailing comma here

Reply:
done